.github/workflows/run-native-install.yml

```diff
@@ -0,0 +1,15 @@
+on: workflow_dispatch
+
+jobs:
+  native_install:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Run native install
+        env:
+          OPENEDX_RELEASE: open-release/lilac.master
+        run: |
+          echo -e "EDXAPP_LMS_BASE: '$(curl ipinfo.io/ip)'\nEDXAPP_CMS_BASE: '$(curl ipinfo.io/ip):18010'" > config.yml
+          wget https://raw.githubusercontent.com/edx/configuration/$OPENEDX_RELEASE/util/install/ansible-bootstrap.sh -O - | sudo -E bash
+          wget https://raw.githubusercontent.com/edx/configuration/$OPENEDX_RELEASE/util/install/generate-passwords.sh -O generate-password.sh | bash
+          wget https://raw.githubusercontent.com/edx/configuration/$OPENEDX_RELEASE/util/install/native.sh -O - | bash
```

Comment:
Currently getting this failure

`TASK [mysql : Look for mysql 5.6] **********************************************
ok: [localhost]

TASK [mysql : Important message] ***********************************************
skipping: [localhost]

TASK [mysql : pause] ***********************************************************
skipping: [localhost]

TASK [mysql : Stop mysql service] **********************************************
skipping: [localhost]

TASK [mysql : Remove experimental apt repository] ******************************
skipping: [localhost]

TASK [mysql : Remove experimental version of mysql] ****************************
[DEPRECATION WARNING]: Invoking "apt" only once while using a loop via 
squash_actions is deprecated. Instead of using a loop to supply multiple items 
and specifying `name: "{{ item }}"`, please use `name: ['mysql-server-5.6', 
'mysql-server']` and remove the loop. This feature will be removed in version 
2.11. Deprecation warnings can be disabled by setting 
deprecation_warnings=False in ansible.cfg.
skipping: [localhost] => (item=[]) 

TASK [mysql : Install mysql dependencies] **************************************
skipping: [localhost]

TASK [mysql : Install mysql] ***************************************************
skipping: [localhost]

TASK [mysql : Set default character sets and collations] ***********************
skipping: [localhost]

TASK [mysql : add the mysql signing key] ***************************************
skipping: [localhost]

TASK [mysql : add the mysql-5.7 repo to the sources list] **********************
skipping: [localhost]

TASK [mysql : install mysql-5.7-client] ****************************************
skipping: [localhost]

TASK [mysql : install mysql-5.7-community-server] ******************************
skipping: [localhost]

TASK [mysql : install mysql-5.7-server] ****************************************
skipping: [localhost]

TASK [mysql : restart mysql] ***************************************************
skipping: [localhost]

TASK [mysql : Ensure Anonymous user(s) does not exist] *************************
skipping: [localhost] => (item=localhost) 
skipping: [localhost] => (item=fv-az137-97) 

TASK [mysql : Alter user root to use mysql_native_password] ********************
skipping: [localhost]

TASK [edxlocal : Install packages needed for single server] ********************
[DEPRECATION WARNING]: Invoking "apt" only once while using a loop via 
squash_actions is deprecated. Instead of using a loop to supply multiple items 
and specifying `name: "{{ item }}"`, please use `name: '{{ edxlocal_debian_pkgs
 }}'` and remove the loop. This feature will be removed in version 2.11. 
Deprecation warnings can be disabled by setting deprecation_warnings=False in 
ansible.cfg.
changed: [localhost] => (item=['postfix', 'libjpeg-dev'])

TASK [edxlocal : create databases] *********************************************
failed: [localhost] (item=ecommerce) => {"ansible_loop_var": "item", "changed": false, "item": "ecommerce", "msg": "unable to find /home/runner/.my.cnf. Exception message: (2002, \"Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)\")"}
failed: [localhost] (item=dashboard) => {"ansible_loop_var": "item", "changed": false, "item": "dashboard", "msg": "unable to find /home/runner/.my.cnf. Exception message: (2002, \"Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)\")"}
failed: [localhost] (item=xqueue) => {"ansible_loop_var": "item", "changed": false, "item": "xqueue", "msg": "unable to find /home/runner/.my.cnf. Exception message: (2002, \"Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)\")"}
failed: [localhost] (item=edxapp) => {"ansible_loop_var": "item", "changed": false, "item": "edxapp", "msg": "unable to find /home/runner/.my.cnf. Exception message: (2002, \"Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)\")"}
failed: [localhost] (item=edxapp_csmh) => {"ansible_loop_var": "item", "changed": false, "item": "edxapp_csmh", "msg": "unable to find /home/runner/.my.cnf. Exception message: (2002, \"Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)\")"}
failed: [localhost] (item=edx_notes_api) => {"ansible_loop_var": "item", "changed": false, "item": "edx_notes_api", "msg": "unable to find /home/runner/.my.cnf. Exception message: (2002, \"Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)\")"}
failed: [localhost] (item=analytics-api) => {"ansible_loop_var": "item", "changed": false, "item": "analytics-api", "msg": "unable to find /home/runner/.my.cnf. Exception message: (2002, \"Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)\")"}
failed: [localhost] (item=reports) => {"ansible_loop_var": "item", "changed": false, "item": "reports", "msg": "unable to find /home/runner/.my.cnf. Exception message: (2002, \"Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)\")"}
failed: [localhost] (item=blockstore) => {"ansible_loop_var": "item", "changed": false, "item": "blockstore", "msg": "unable to find /home/runner/.my.cnf. Exception message: (2002, \"Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)\")"}
skipping: [localhost] => (item=) 
failed: [localhost] (item=discovery) => {"ansible_loop_var": "item", "changed": false, "item": "discovery", "msg": "unable to find /home/runner/.my.cnf. Exception message: (2002, \"Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)\")"}
skipping: [localhost] => (item=) 
skipping: [localhost] => (item=) 

PLAY RECAP *********************************************************************
localhost                  : ok=58   changed=30   unreachable=0    failed=1    skipped=195  rescued=0    ignored=0   

 
============================================================
Ansible failed!
------------------------------------------------------------
 
Decoded error:
No message to decode.
 
============================================================`

Reply:
Looks like the runner comes with a Mysql 8.0 installed with a root password. https://github.com/actions/virtual-environments/blob/ubuntu20/20210412.1/images/linux/Ubuntu2004-README.md#mysql